xml/upgrade-3.10-4.xsl

```diff
@@ -69,4 +73,13 @@
     </xsl:call-template>
 </xsl:template>
 
+<!-- Drop restart-type resource meta-attribute -->
```

Comment:
We could also drop `CRM_meta_restart-type` as an instance attribute, since that would have the same effect as a `restart-type` meta-attribute... it wouldn't be that much more code but I slightly lean against the clutter. It's unfortunate that we have that little loophole, where a `CRM_meta_X` instance attribute is the same as an `X` meta-attribute.